docs/reference/style_guide.md

```diff
@@ -1,18 +1,33 @@
 
-Our ESLint rules are run on Codacy, to alert of errors there, but only there, as to not interfere with the development practice. Having them warned of there gives us the ability to see where our Style Guide is being broken and fix it as needed, while not hounding a new Contributor with these issues.
+The following list contains all coding style guide rules that should be adhered to:
 
-## Style Guide Rules
+  * `camelCase`: Usage of `camelCase` is enforced to keep variable names readable, and keeping code fragments in sync. (Enforced via `ESLint`)
 
-* camelCase: We enforce the use of camelCase variables within this repo for readability purposes, and keeping the code similar enough throughout the codebase. This rule is enforced via ESLint.
-* Cyclomatic Complexity: We enforce a maximum cyclomatic complexity on any one function to reduce the likelihood of mistakes and bugs within a code block. This rule is enforced via ESLint.
-* Type-Safe Equality Operators: We enforce the use of `===` or `!==` to avoid type coercion. Some instances it will be needed to use alternative comparisons, and these are accounted for. This rule will not flag usage when comparing against two literal values, when evaluating `typeof`, and when comparing against `null`. This rule is enforced via ESLint.
-* Max Nesting: We enforce a Max Nesting Depth of `4` within a code block. Following our coding principle of [Early Return Patterns](ARCHITECTURE.md). This is enforced via ESLint.
-* Double Quotes: We enforce the usage of Double Quotes when possible within code. This rule is enforced via Prettier and should be done automatically once code is merged into the repo.
-* Semicolons: We enforce the use of Semicolons at the end of lines. This rule is enforced via Prettier.
+  * Cyclomatic Complexity: The cyclomatic complexity of any individual function is monitored to reduce the likely-hood of error prone code, due to complextiy. (Enforced via `ESLint`)
+
+  * Type-Safe Equality Operators: Usage of equality operators is enforced to avoid type coercion. Using a Type-Safe Equality Operator refers to using `===` or `!===` instead of `==` and `!=`. Some instances do require using alternative comparison methods, which are accounted for, such as `typeof` and `null` comparisons. That is, any comparisons between two literal values will not be enforced or flagged. (Enforced via `ESLint`)
+
+  * Max Nesting: This code base enforces a Max Nesting depth of `4` within a single code block. Enforcing a max nesting depth helps to ensure adherence to this repositories coding principle [Early Return Pattern](ARCHITECTURE.md). (Enforced via `ESLint`)
+
+  * Double Quotes: Usage of double quotes instead of single quotes is enforced within this repository. (Enforced via `prettier`)
+
+  * Semicolons: Ensuring to end each line/block of JavaScript with a semicolon is required. (Enforced via `prettier`)
+
+## Documentation Style Guide
```

Comment:
Ideally we should have a global doc for this we can link to instead. We have one I wrote a long time ago in the website "resources" docs. https://github.com/pulsar-edit/pulsar-edit.github.io/blob/main/docs/docs/resources/website/sections/document-style.md (although that was made for the main docs specifically), maybe we need to look at making a master one in `.github`?

Reply:
Really if we could, it'd be awesome to have all of our docs following a guideline, including those on the website, and those for each repo individually. So I would be more than down to create one within `.github` would just be a matter of defining them, since as of now, there's not a whole lot of thought going into style beyond here and the website itself